demos/_assets/css/jqm-demos.css

```diff
@@ -518,6 +518,14 @@
 	opacity: .75;
 }
 
+/* Darker hover state for links. Overwriting jquery.mobile.theme.css LN298 */
```

Comment:
Remove line number (theme CSS isn't static so the line number will change sooner or later).